aries-site/src/themes/aries.js

```diff
@@ -6,20 +6,23 @@ export const aries = deepMerge(hpe, {
   global: {
     colors: {
       focus: 'teal!',
-      placeholder: 'text-xweak',
```

Comment:
You might not be up to date with master here, but I don't think we want to remove this block.